launch/launch/event_handler.py

```diff
@@ -48,16 +52,63 @@ def __init__(
             the event should be handled by this event handler, False otherwise.
         :param: entities is an LaunchDescriptionEntity or list of them, and is
             returned by handle() unconditionally if matcher returns True.
+        :param: handle_once is a flag that, if True, unregisters this EventHandler
+            after being handled once.
         """
         self.__matcher = matcher
         self.__entities = entities
+        self.handle_once = handle_once
 
     @property
     def entities(self):
         """Getter for entities."""
+        # if self.__entities is None:
+        #     return []
         return self.__entities
 
-    # TODO(wjwwood): setup standard interface for describing event handlers
+    @property
+    def handle_once(self):
+        """Getter for handle_once flag."""
+        return self.__handle_once
+
+    @handle_once.setter
+    def handle_once(self, value):
+        """Setter for handle_once flag."""
```

Comment:
might be wise to remove the public setter so that users can't get into strange situations by modifying the flag. FWICT it'd be sufficient to just set it in the constructor now. If a need comes up later, we can consider re-adding it to the API (it's harder though to _remove_ API).